package.json

```diff
@@ -13,27 +13,14 @@
   "scripts": {
     "postinstall": "electron-builder install-app-deps && rimraf node_modules/dtrace-provider",
     "start": "electron .",
-    "asarl": "asar l release/mac/Signal.app/Contents/Resources/app.asar",
+    "grunt": "grunt",
     "icon-gen": "electron-icon-maker --input=images/icon_1024.png --output=./build",
     "generate": "npm run icon-gen && grunt",
     "build": "build  --config.extraMetadata.environment=$SIGNAL_ENV",
-    "dist": "npm run generate && npm run build",
-    "pack": "npm run generate && npm run build -- --dir",
+    "build-release": "SIGNAL_ENV=production npm run build -- --config.directories.output=release",
     "prepare-beta-build": "node prepare_beta_build.js",
     "prepare-import-build": "node prepare_import_build.js",
-    "pack-prod": "SIGNAL_ENV=production npm run pack",
-    "dist-prod": "SIGNAL_ENV=production npm run dist",
-    "dist-prod-all": "SIGNAL_ENV=production npm run dist -- -mwl",
-    "build-release": "SIGNAL_ENV=production npm run build -- --config.directories.output=release",
-    "build-mas-release": "npm run build-release -- -m --config.mac.target=mas",
-    "build-mas-dev": "npm run build-release -- -m --config.mac.target=mas --config.type=development",
-    "grunt": "grunt",
-    "prep-mac-release": "npm run build-release -- -m --dir",
-    "prep-release": "npm run generate && grunt prep-release && npm run build-release && npm run build-mas-release && grunt test-release",
-    "release-mac": "npm run build-release -- -m --prepackaged release/mac/Signal*.app --publish=always",
-    "release-win": "npm run build-release -- -w --prepackaged release/windows --publish=always",
-    "release-lin": "npm run build-release -- -l --prepackaged release/linux && NAME=$npm_package_name VERSION=$npm_package_version ./aptly.sh",
-    "release": "npm run release-mac && npm run release-win && npm run release-lin",
```

Comment:
Wow! 🔪

Can you explain a bit more on how builds are now done? Does this mean `yarn build-release` will now make a platform-specific build depending on the OS it runs?

Reply:
`yarn build-release` is a base task on top of which we'll write other scripts to do the actual build. For example, my OSX build script has this line in it: `yarn build-release -m`

It used to be that the `release-xxx` tasks would actually upload to S3 for us, but I've moved that out into my release scripts so we can re-upload if the upload fails - _without_ re-building!

Reply:
Thanks for elaborating on this 😄